guides/lagom-dcos-marathon-deploy-microservices/src/main/paradox/index.md

```diff
@@ -0,0 +1,338 @@
+Marathon-lb DC/OS Service has been successfully installed!
+See https://github.com/mesosphere/marathon-lb for documentation.
+```
+
+## 4. Build Chirper Docker images (Optional)
+
+Chirper is configured to build Docker images for each of its microservices. This is accomplished with both sbt and 
+Maven build tools, covered below.
+
+_Lightbend has built and published these images to Bintray. If you are using the default configuration, you may
+choose to skip this step._
+
+> For general assistance on setting up your Lagom build please refer to ["Defining a Lagom Build" in the Lagom documentation](https://www.lagomframework.com/documentation/1.3.x/scala/LagomBuild.html).
+
+----------------------------------
+
+###### Maven
+
+By using 
+[fabric8's docker-maven-plugin](https://dmp.fabric8.io/), these images will be built and published to the local Docker
+repository. The command below will build Chirper and the Docker images using Maven and this plugin.
+
+> Note that if you see a `[ERROR] DOCKER> Unable to pull...` error with the following then you'll need to update your Java version [as per a known issue with Java TLS](https://github.com/fabric8io/docker-maven-plugin/issues/845#issuecomment-324249997).
+
+<pre class="code-bash prettyprint prettyprinted">
+mvn -DbuildTarget=marathon clean package docker:build
+</pre>
+
+_Refer to the various `pom.xml` files in the [Chirper repository](https://github.com/lagom/activator-lagom-java-chirper) for more details._
+
+###### sbt
+
+By using [sbt native packager](https://github.com/sbt/sbt-native-packager) Chirper is configured
+to be able to build Docker images. The command below will build Chirper and the Docker images using
+sbt and this plugin. Since Chirper can be built for many orchestration platforms, you must specify
+the `-DbuildTarget=marathon` option. Your own project can be streamlined if you only wish to target DC/OS.
+
+<pre class="code-bash prettyprint prettyprinted">
+sbt -DbuildTarget=marathon clean docker:publishLocal
+</pre>
+
+_Refer to `build.sbt` in the [Chirper repository](https://github.com/lagom/activator-lagom-java-chirper) for more details._
+
+----------------------------------
+
+Next, inspect the images that are available. Note that the various Chirper services all have their own image. These will
+be deployed to the cluster.
+<pre class="code-bash prettyprint prettyprinted">
+docker images
+</pre>
+
+```
+REPOSITORY                             TAG              IMAGE ID      CREATED       SIZE
+chirper-marathon/front-end             1.1.14-SNAPSHOT  65837188dde1  20 hours ago  767MB
+chirper-marathon/friend-impl           1.1.14-SNAPSHOT  a8a589c6c16f  20 hours ago  845MB
+chirper-marathon/chirp-impl            1.1.14-SNAPSHOT  d3e32819f040  20 hours ago  844MB
+chirper-marathon/activity-stream-impl  1.1.14-SNAPSHOT  072cb98868d5  20 hours ago  760MB
+```
+
+## 5. Tag and Push Chirper Docker images (Optional) 
```

Comment:
Suggest combining this into the previous step, since it only makes sense if you have already built the images locally.